app/crud.py

```
@@ -0,0 +1,152 @@
+from fastapi import HTTPException
+from sqlalchemy.exc import NoResultFound
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+
+from .models import ToDo
+
+
+async def create_todo(db: AsyncSession, todo: ToDo):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_todo`

(ANN201)